lib/ansible/module_utils/eos.py

```diff
@@ -180,6 +191,9 @@ def _load_params(self):
             if key in NET_COMMON_ARGS.keys():
                 if not params.get(key) and value is not None:
                     params[key] = value
+        for key, env_var in NET_ENV_ARGS.items():
+            if not params.get(key) and os.environ.get(env_var):
```

Comment:
This should ask if params.get(key) is None.  That way you can tell that the user didn't specify a falsey value (like authorize=False).